rustic-doc.el

```diff
@@ -287,14 +288,17 @@ If the user has not visited a project, returns the main doc directory."
          (proc (let ((process-connection-type nil))
                  (apply #'start-process name buf program program-args))))
     (set-process-sentinel
-     proc (lambda (proc _event)
+     proc (lambda (proc event)
             (let ((buf (process-buffer proc)))
-              (when finish-func
-                (funcall finish-func proc))
-              (when (buffer-live-p buf)
-                (kill-buffer buf)))))
+              (if (string-match-p (regexp-quote "abnormally") event)
+                  (message "Could not finish process: %s. See the *Messages* buffer or %s for more info." event (concat "*" name "*"))
```

Comment:
You can't access `name` from the sentinel.

Reply:
I try like this: 
* Add `exit 1` to the top of `/home/sam/.local/bin/rustic-doc-convert.sh`.
* Comment out the line ` (rustic-doc--install-resources)` in `rustic-doc-setup` inside `rustic-doc.el` (otherwise the script would * run `rustic-doc-setup`
This gives the message
`Could not finish process: exited abnormally with code 1
. See the *Messages* buffer or *rustic-doc-std-conversion* for more info.`

So it seems to me that I can access the name in the sentinel, or am I misunderstanding something?

Reply:
Ah you're right. You can capture the variable if you use a lambda. If you would use a function instead of the lambda it wouldn't work. I'm not coding much in elisp for some time.

Reply:
It's good to make sure. Thank you!